src/identity/components/Avatar.tsx

```diff
@@ -1,13 +1,19 @@
+import type { Address } from 'viem';
+import { useEffect, useState } from 'react';
+import { base } from 'viem/chains';
+import { getEASAttestations } from '../getEASAttestations';
 import { useAvatar } from '../hooks/useAvatar';
 import { useName } from '../hooks/useName';
-import type { Address } from 'viem';
+import { useVerified } from '../hooks/useVerified';
+import { WithAvatarBadge } from './internal/WithAvatarBadge';
 
 type AvatarProps = {
   address: Address;
   className?: string;
   loadingComponent?: JSX.Element;
   defaultComponent?: JSX.Element;
   props?: React.ImgHTMLAttributes<HTMLImageElement>;
+  displayBadge?: boolean;
```

Comment:
I am not super sure `displayBadge` is the right name. I mean is not bad. @kyhyco what's your take?

Reply:
some other options:
- `displayAttestation`
- `displayAttested`
- `showBadge`
- `isAttested`

Reply:
How about just `attest`?